pythonScripts/glucoseDataframe.py

```diff
@@ -6,8 +6,8 @@
 import pandas as pd
 import numpy as np
 import os
+from datetime import *
```

Comment:
Any reason you did this rather than just import datetime?

Reply:
It doesn't let me use strptime with just import datetime, which does not make sense?

Reply:
hm weird. You could probably do
```python
import datatime
from datatime import strptime
```
but I wouldn't bother that looks arguably worse.